proxy/proxy.go

```diff
@@ -135,14 +135,14 @@ func NewProxy(proxyHandler Handler, caKeyPath, caCertPath string, insecureSkipVe
 	return &Proxy{ProxyHttpServer: proxy}, nil
 }
 
-func NewReverseProxy(vefiyingHandler Handler, proxyHandler Handler, authHandler Handler, authServicePath string, excludes... *regexp.Regexp) (*Proxy, error) {
+func NewReverseProxy(verifyingHandler Handler, proxyHandler Handler, authHandler Handler, authServicePath string, excludes ...*regexp.Regexp) (*Proxy, error) {
 	// Create a reverse proxy.
 	reverseProxy := goproxy.NewReverseProxyHttpServer()
 	reverseProxy.Tr = http.DefaultTransport.(*http.Transport)
 	reverseProxy.Verbose = log.GetLevel() == log.DebugLevel
 
 	// Handle requests with the validation handler.
-	reverseProxy.OnRequest(goproxy.Not(IsCorsPreflight()), goproxy.Not(UrlMatches(excludes...))).DoFunc(vefiyingHandler)
+	reverseProxy.OnRequest(goproxy.Not(IsCorsPreflight()), goproxy.Not(UrlMatches(excludes...))).DoFunc(verifyingHandler)
 
 	if authServicePath != "" {
```

Comment:
Shouln't we add cookiesEnabled here also ?

Reply:
I believe they are not really depending. Cookies authentication may be used independently from the authentication endpoint.

Reply:
Now, I get what you mean. So, I think It makes sense to check if `authServicePath` is not empty and   `cookiesEnabled` is false that it is kind of wrong configuration. I'll consider printing a warning message.